api/router.go

```diff
@@ -1,6 +1,7 @@
 package api
 
 import (
+	"github.com/longhorn/longhorn-manager/monitoring"
```

Comment:
add new section after `apierrors "k8s.io/apimachinery/pkg/api/errors"` a new line acts as a separator between the sections.

Reply:
Thank you! Fixed